feat: Add docs for self-referral traffic (#9408)
PostHog/posthog.com: contents/docs/web-analytics/faq.mdx

* Websites can specify that the referrer should not be sent, by including the `referrerpolicy` meta tag in the page or `rel="noreferrer"` on the link.

> **Note:** If you have auto-tagged traffic set up in Google Ads, parameters set this way do not appear as URL parameters, and therefore aren’t available for us to use for attribution. You can set up tracking templates in Google Ads to add query parameters automatically.

## Why does my own site appear as a referrer?

This is called self-referral traffic, having a small amount (<5%) of self-referral traffic is normal, and it can happen for a few reasons:

> ### Cross-domain tracking

Reviewer: ```suggestion
### Cross-domain tracking

```

If you track multiple domains, you might need to set up [cross-domain tracking](/tutorials/cross-domain-tracking) to ensure that sessions are correctly attributed.

> ### Session timeout

Reviewer: ```suggestion
### Session timeout

```

1. A user visits a site (for example, `google.com`) that links to yours.

> 2. The user clicks a link to your site. At this point a new session will be started and the referrer will be `google.com`.

Reviewer: ```suggestion
2. The user clicks a link to your site. At this point, a new session starts and the referrer is `google.com`.
```

3. The user walks away from their computer, and is inactive long enough for the session to expire (30 minutes by default).

> 4. User comes back to their computer, goes to the already-open browser tab with your site, and clicks an internal link. A new session will be started, and the referrer will be your own site.

Reviewer: ```suggestion
4. User comes back to their computer, goes to the already-open browser tab with your site, and clicks an internal link. A new session starts and the referrer is your own site.
```
